test/e2e_node/conformance_test.go

```diff
@@ -62,7 +62,7 @@ var _ = Describe("Container Conformance Test", func() {
 				phase := api.PodPending
```

Comment:
From [line 58](https://github.com/kubernetes/kubernetes/pull/23285/files#diff-803e7d016e077a89c74759ca80d447ceR58): It seems more like this is testing that the container stopped correctly?